apache/superset
# feat(explore): Frontend implementation of dataset creation from infobox

### SUMMARY

This is the front end implementation of dataset creation from an infobox in a chart.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### BEFORE:
![cdsfibBefore](https://user-images.githubusercontent.com/55605634/165389134-1ec551e2-50ab-4dba-95e7-cfcd7d1015ca.png)

#### AFTER:
![cdsfibAfter](https://user-images.githubusercontent.com/55605634/165407681-ec87570f-0530-47ec-8d3e-79ae8ce2deea.png)
![saveDSmodal](https://user-images.githubusercontent.com/55605634/165407788-436be23e-2bc9-45f5-8ab3-5602645574c5.png)


### TESTING INSTRUCTIONS

- Open a chart with a query datasource type 
- Observe the new infobox on the left panel
- Click "Create a dataset" and observe the modal
- Close the modal and close the infobox
- Refresh the page and observe that the infobox does not return
- Close the browser tab containing this instance of Superset and reopen a new one
- Observe that the infobox has returned on a new session

### ADDITIONAL INFORMATION
